Sources/PackageGraph/PackageGraphRoot.swift

```diff
@@ -48,8 +48,8 @@ public struct PackageGraphRoot {
     /// Create a package graph root.
     public init(input: PackageGraphRootInput, manifests: [Manifest], explicitProduct: String? = nil) {
         self.packageRefs = zip(input.packages, manifests).map { (path, manifest) in
-            let identity = PackageIdentity(url: manifest.url)
-            return PackageReference(identity: identity, path: path.pathString, kind: .root)
+            let identity = PackageIdentity(url: manifest.packageLocation)
```

Comment:
This also seems wrong, root packages should use the `path` initializer.

Reply:
yup https://github.com/apple/swift-package-manager/pull/3155/commits/dc768db5dbc32ea8f183c3d725a31877e2736ad7#r552146856